node/api.go

```diff
@@ -260,6 +260,27 @@ func (api *PrivateAdminAPI) StopWS() (bool, error) {
 	return true, nil
 }
 
+// GetAllDiscoveryBuckets gives the content of all buckets in the p2p discovery table
+func (api *PrivateAdminAPI) GetAllDiscoveryBuckets() ([]interface{}, error) {
```

Comment:
I think having a specific informational struct for the table (e.g. `discTableInfo`) would be more aligned to how the current code works.  E.g. there is a `peerInfo` and `nodeInfo` that the admin API uses.

https://github.com/celo-org/celo-blockchain/blob/master/p2p/peer.go#L443
https://github.com/celo-org/celo-blockchain/blob/master/p2p/server.go#L1190